docs/attempts-api/schemas/events/identity-proofing/IdvIppReadyToVerifyVisit.yml

```diff
@@ -1,5 +1,5 @@
 description: |
-  When the user submits the security code that they received on their phone to verify their phone or address during identity proofing.
+  When the user transfers to the in-person proofing workflow.
```

Comment:
I'd recommend updating this comment to clarify that the user is already in the in-person proofing process when they visit the IPP ready to verify page.

Reply:
updated in https://github.com/18F/identity-idp/pull/12093/commits/23872c5ee580b9e75b3d1466547ed5f4bc9180f3